build.gradle

```diff
@@ -58,9 +60,19 @@ sourceSets {
         }
     }
 }
+cordapp {
+    info {
+        name "CorDapp Option"
+        vendor "r3"
```

Comment:
I thought we agreed to keep `vendor` set to the "original" value.

Reply:
done